backend/main.development.js

```diff
@@ -43,6 +42,7 @@ app.on('ready', () => {
     httpsServer.electronWindow = mainWindow;
 
     const HTTP_URL = `${httpServer.protocol}://${httpServer.domain}:${httpServer.port}`;
+    // TODO ESLint no-unused-vars: though not used, would it be better to use HTTPS?
     const HTTPS_URL = `${httpsServer.protocol}://${httpsServer.domain}:${httpsServer.port}`;
```

Comment:
hm, if this isn't used then we can remove it. HTTPS happens in a variety of different ways

Reply:
Thx, will remove it.